src/io/hdfs_filesys.cc

```diff
@@ -1,10 +1,10 @@
 // Copyright by Contributors
-#include <algorithm>
-#include <limits>
+#include "./hdfs_filesys.h"
```

Comment:
Thanks, I will remove it and possibly submit another PR to fix the lint error on Travis CI.